src/posts/models.py

```diff
@@ -26,8 +26,20 @@ class Post(Base):  # pylint: disable=too-few-public-methods
     topic_id: int = Column(Integer, ForeignKey("topics.id"), nullable=False)
     author: User = relationship("User", uselist=False)
 
+    SORTING_FIELDS: Final = ("created_at")
+    SORTING_ORDER: Final = ("asc", "desc")
+
     @staticmethod
-    def get_post_by_id(topic_id: int) -> Post | None:
+    def get_post_by_id(post_id: int) -> Post | None:
```

Comment:
Let's make this change in a separate ticket I created: https://github.com/fenya123/forum123/issues/114